File: NOVEMBER-2025/24-11-2025/1018.swift
class Solution {
    func prefixesDivBy5(_ nums: [Int]) -> [Bool] {
        var ans: [Bool] = []
        var mod = 0

        for bit in nums {
            mod = (mod * 2 + bit) % 5
            ans.append(mod == 0)
        }

        return ans
    }
}

File: NOVEMBER-2025/24-11-2025/1018.php
class Solution {

    /**
     * @param Integer[] $nums
     * @return Boolean[]
     */
    function prefixesDivBy5($nums) {
        $ans = [];
        $mod = 0;

        foreach ($nums as $bit) {
            $mod = ($mod * 2 + $bit) % 5;
            $ans[] = ($mod == 0);
        }

        return $ans;
    }
}